src/efjm.tw2

```diff
@@ -7,9 +7,16 @@ Escape from Juggalo Mountain
 [[Begin|Jail Cell (Unlit) - name character]]
 [[Start from Outside Port-a-potty|Outside Port-a-potty]]
 [[Opening Crawl|Opening crawl]]
-[[Mom's Tent|Mom's Tent - name prompt]]
-[[Jump to Mom's Tent - table|Mom's Tent - name prompt table]]
-[[Jump to Mom's Tent - letter|Mom's Tent - name prompt letter]]
+[[Start from Mom's Tent - table|Mom's Tent - name prompt table]]
+[[Start from Mom's Tent - letter|Mom's Tent - name prompt letter]]
+[[Start from the Chase|Tent City - Intro]]
+- [[Chase 1: Three Rings Recruiter|Tent City - Three Rings recruiter]]
```

Comment:
Not sure we need this debug link. Looks like the first Tent City passage sets variables that you need to see all the choices in the Three Rings passage. So, if you use this link, you only see the two correct choices.